package.json

```diff
@@ -15,8 +15,9 @@
     "build": "gatsby build --prefix-paths",
     "serve-build": "gatsby serve --prefix-paths",
     "update-icons": "git submodule update --init --remote && node scripts/createIconTypes.js && node scripts/createIconNames.js",
-    "build-pdf": "python3 scripts/pdf/generate_pdf.py",
-    "build-all-pdfs": "for i in product_docs/docs/**/*/ ; do echo \"$i\"; python3 scripts/pdf/generate_pdf.py ${i%} || exit 1; done",
+    "build-pdf": "docker-compose -f docker/docker-compose.build-pdf.yaml run --rm --entrypoint scripts/pdf/generate_pdf.py pdf-builder",
+    "build-all-pdfs": "for i in product_docs/docs/**/*/ ; do echo \"$i\"; yarn build-pdf ${i%} || exit 1; done",
```

Comment:
Curious how much overhead there realistically is in calling `yarn build-pdf` in a loop, now that it's dockerized.

Reply:
In my very unscientific testing:

* `build-all-pdfs-ci`: 8 minutes
* `build-all-pdfs`: 10 minutes

So I feel it's acceptable.

Reply:
Yeah that seems fine, was just curious